w3c.json

```
@@ -0,0 +1,5 @@
+{
+  "group": 93489,
+  "contacts": ["johnluther", "mavgit", "tidoust"],
```

Since Mark won't be editing this anymore, we should probably swap out his name for mine
```suggestion
  "contacts": ["johnluther", "johnriv", "tidoust"],
```